build.mill.scala

```diff
@@ -1344,6 +1344,11 @@ object `local-repo` extends LocalRepo {
 
 // Helper CI commands
 def publishSonatype(tasks: mill.main.Tasks[PublishModule.PublishData]) = Task.Command {
+  val taskNames = tasks.value.map(_.toString())
+  System.err.println(
+    s"""Tasks producing artifacts to be included in the bundle:
```

Comment:
We could get the suze around here, no?

Reply:
nope... the actual bundle is created much later, inside of Mill API... 😕